litmus-portal/frontend/src/views/Analytics/ApplicationDashboard/ChaosAccordion/index.tsx

```diff
@@ -57,68 +58,97 @@ const StyledAccordionDetails = withStyles((theme) => ({
-      >
-        <TextButton
-          className={classes.button}
-          onClick={() => setChaosTableOpen(!chaosTableOpen)}
-          variant="highlight"
-          startIcon={
-            !chaosTableOpen ? (
-              <ArrowDropDownIcon className={classes.tableDropIcon} />
-            ) : (
-              <ArrowDropUpIcon className={classes.tableDropIcon} />
-            )
-          }
+    <>
+      <Accordion expanded={chaosTableOpen}>
+        <AccordionSummary
+          aria-controls="chaos-table-content"
+          id="chaos-table-header"
+          className={classes.accordionSummary}
+          key={`chaos-table-${dashboardKey}`}
         >
-          <Typography className={classes.chaosHelperText}>
-            {!chaosTableOpen
-              ? t(
-                  'analyticsDashboard.monitoringDashboardPage.chaosTable.showTable'
-                )
-              : t(
-                  'analyticsDashboard.monitoringDashboardPage.chaosTable.hideTable'
-                )}
-          </Typography>
-        </TextButton>
-        <IconButton
-          aria-label="edit chaos query"
-          aria-haspopup="true"
-          disabled
-          data-cy="editChaosQueryButton"
-          className={classes.editIconButton}
-        >
-          <img src="/icons/editIcon.svg" alt="Edit" />
-        </IconButton>
-      </AccordionSummary>
-      <StyledAccordionDetails className={classes.accordionDetails}>
-        <ChaosTable
-          chaosList={chaosEventsToBeShown}
-          selectEvents={(selectedEvents: string[]) =>
-            postEventSelectionRoutine(selectedEvents)
-          }
-        />
-      </StyledAccordionDetails>
-    </Accordion>
+          <TextButton
+            className={classes.button}
+            onClick={() => setChaosTableOpen(!chaosTableOpen)}
+            variant="highlight"
+            startIcon={
+              !chaosTableOpen ? (
+                <ArrowDropDownIcon className={classes.tableDropIcon} />
+              ) : (
+                <ArrowDropUpIcon className={classes.tableDropIcon} />
+              )
+            }
+          >
+            <Typography className={classes.chaosHelperText}>
+              {!chaosTableOpen
+                ? t(
+                    'analyticsDashboard.monitoringDashboardPage.chaosTable.showTable'
+                  )
+                : t(
+                    'analyticsDashboard.monitoringDashboardPage.chaosTable.hideTable'
+                  )}
+            </Typography>
+          </TextButton>
+          <IconButton
+            aria-label="edit chaos query"
+            aria-haspopup="true"
+            onClick={() => setChaosAnnotationsEditorOpen(true)}
+            data-cy="editChaosQueryButton"
+            className={classes.editIconButton}
+          >
+            <img src="/icons/editIcon.svg" alt="Edit" />
+          </IconButton>
+        </AccordionSummary>
+        <StyledAccordionDetails style={{ width: '100%' }}>
```

Comment:
Can we remove inline styles?

Reply:
since we are using only single property using a class will be an overkill